server/src/useCases/report/productCompatibility/handleGetProductRequest.js

```diff
@@ -9,10 +11,15 @@ module.exports = class ReportGetProductRequestHandler {
   }
 
   async getReports(repository) {
-    repository.aggregateByProduct({}, (err, result) => {
+    const limit = FiltersValidation.validateFilter(this.req.query.limit, Infinity);
+    const offset = FiltersValidation.validateFilter(this.req.query.offset, 0);
```

Comment:
What is behavior when offset exceeds number of entries in database? 
Should we pass count and nextOffset as a part of the payload or is it redundant?

Reply:
From what I tested doesn't show results

Reply:
We agreed with Karolina that she will check if there is more records to get, so I think it's not necessary. What do you think @dborowiecki ?